data/man/kvirc.en.1

```diff
@@ -64,14 +64,14 @@ Use this port for connection.
 This option requires also a server name.
 .TP 8
 .B [ircurl]
-An url in the following form:
+An URL in the following form:
 
   irc[6]://<server>[:<port>][/<channel>[?<pass>]]
 
-For each url on the commandline create a new irc-context
+For each URL on the commandline create a new irc-context
 and attempt to connect to <server> on the specified <port>.
 The optional <channel> is joined just after the connection
-has been estabilished.
+has been established.
```

Comment:
There's no pint in doing the po and pot files they will get overwritten.at regeneration.

Also it will make my job 100 times more difficult.